_sass/components/_c-section-title.scss

```diff
@@ -48,14 +42,8 @@
     margin-bottom: 0;
   }
 
-  + p {
-    font-size: rem(16px);
-    line-height: rem(26px);
-    margin-top: rem(20px);
-  }
-
   @media screen and (min-width: $screen-s) {
-    padding-bottom: rem(50px);
+    padding-bottom: rem(0px);
```

Comment:
0 is 0 in any unit. It's not necessary use the rem function here, just type 0 :)